README.md

```diff
@@ -131,20 +131,21 @@ To index your content from WordPress:
 1. Add your WordPress URL in place of wordpress.domain on line 1 of `.env`.
 
 
-### Rate Checker
-The Rate Checker is a JavaScript application for checking mortgage interest rates. Currently owning-a-home's Rate Checker is powered by two private APIs that returns mortgage rate and county data. **Without these APIs configured, the website will still load but the Rate Checker application will NOT be available.**
+### Rate Checker and Mortgage Insurance
+The Rate Checker and Mortgage Insurance are JavaScript applications for checking mortgage interest rates and mortgage insurance premium. Currently owning-a-home's Rate Checker and Loan Comparison are powered by private APIs that returns mortgage rate, county data, and mortgage insurance premium. **Without these APIs configured, the website will still load but the Rate Checker and Loan Comparison applications will NOT be available.**
```

Comment:
premium should be premiums here

Also this is really making me think we need a dummy data version of the site for external contributors